src/pybind/mgr/dashboard/frontend/src/app/shared/services/timer.service.ts

```diff
@@ -21,6 +21,7 @@ export class TimerService {
     return timer(dueTime, refreshInterval, this.ngZone.leave).pipe(
       observeOn(this.ngZone.enter),
       switchMap(next),
+      retry(),
```

Comment:
Not sure if we really need this @alfonsomthd ? Everything else LGTM!

Reply:
I agree that resubscription behaviour does not apply here, so I removed it. If needed, it should be implemented in wrapped function (```next```).